src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/ElasticsearchQueryHelper.java

```diff
@@ -34,7 +35,12 @@ public ElasticsearchQueryHelper(ApplicationParams applicationParams,
 
     public ElasticsearchRequest validateAndConvertRequest(String jsonSearchRequest) {
         rejectBlackListedQuery(jsonSearchRequest);
-        JsonNode searchRequestNode = objectMapperService.convertStringToObject(jsonSearchRequest, JsonNode.class);
+        JsonNode searchRequestNode;
+        try {
+            searchRequestNode = objectMapperService.convertStringToObject(jsonSearchRequest, JsonNode.class);
+        } catch (Exception ex) {
```

Comment:
could you catch exact Json exceptions  rather than generic one.

Reply:
Updated to only catch ServiceExceptions